lc-hax/Scripts/Modules/Possession/Controllers/NutcrackerController.cs

```diff
@@ -9,7 +8,7 @@ enum NutcrackerState {
 internal class NutcrackerController : IEnemyController<NutcrackerEnemyAI> {
     public void OnMovement(NutcrackerEnemyAI enemy, bool isMoving, bool isSprinting) {
         if (isMoving || isSprinting) return;
-        enemy.SwitchToBehaviourServerRpc(Convert.ToInt32(NutcrackerState.WALKING));
+        enemy.SetBehaviourState(NutcrackerState.WALKING);
```

Comment:
i intentionally used SwitchToBehaviourServerRpc because the SetBehaviourState does check if is not in that state, that is what's causing the unwanted sentry mode.

Reply:
Makes no sense to me. If it's already in the walking state, why set it to the walking state again?

Reply:
> Makes no sense to me. If it's already in the walking state, why set it to the walking state again?

to prevent the AI from setting it to random sentry modes, and avoid the animation of it unwantedly.